app/Models/RegisterConfirm.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class RegisterConfirm extends Model
```

Comment:
Why do we need a separate table and model for this if we can have only one hash?